markdown/events/2021/bytesize-25-nf-core-ampliseq.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Bytesize 25: nf-core/ampliseq"
+subtitle: Daniel Straub - QBic, University of Tuebingen, Germany
+type: talk
+start_date: "2021-10-26"
+start_time: "13:00 CEST"
+end_date: "2021-10-26"
+end_time: "13:30 CEST"
+youtube_embed: https://youtu.be/a0VOEeAvETs
+location_url:
+ - https://youtu.be/a0VOEeAvETs
+ - https://zoom.us/j/95310380847
+---
+
+# nf-core/bytesize
+
+Join us for a special pipeline-focussed episode of our **weekly series** of short talks: **“nf-core/bytesize”**.
+
+Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
+These will be recorded and made available at <https://nf-co.re>
+It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
+
+## Bytesize 25: nf-core/ampliseq
+
+This week, Daniel Straub ([@d4straub](https://github.com/d4straub/)) will tell us all about the nf-core/ampliseq pipeline.
```

Comment:
```suggestion
This week, Daniel Straub ([@d4straub](https://github.com/d4straub/)) will tell us all about the nf-core/ampliseq pipeline.

nfcore/ampliseq is a bioinformatics analysis pipeline used for amplicon sequencing, supporting denoising of any amplicon and, currently, taxonomic assignment of 16S, ITS and 18S amplicons. Supported is paired-end Illumina or single-end Illumina, PacBio and IonTorrent data. Default is the analysis of 16S rRNA gene amplicons sequenced paired-end with Illumina.
```

Just copy-pasted from the pipeline docs, but maybe @d4straub has a catchier intro to add a bit more detail to this talk announcement.

Reply:
Good idea! I'll ask him if he can update it closer to the date.